Vagrantfile

```diff
@@ -56,16 +56,16 @@ Vagrant.configure(2) do |config|
     sudo groupadd docker
     sudo usermod -aG docker vagrant
     # Install Docker Compose
-    curl -L https://github.com/docker/compose/releases/download/1.17.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
-    chmod +x /usr/local/bin/docker-compose
+    curl -L https://github.com/docker/compose/releases/download/1.17.0/docker compose-`uname -s`-`uname -m` > /usr/local/bin/docker compose
+    chmod +x /usr/local/bin/docker compose
```

Comment:
This seems like a bug, since it was creating a binary, but that binary now will attempt to have a space in its name.

I would recommend simply deleting the Vagrantfile, personally. If that's too big a change, we could remove this `curl` call that was installing docker-compose, since it's now included as a docker subcommand.

Reply:
Oh, that's an excellent observation. I will just remove the Vagrantfile and anything related to it.